src/OSPSuite.Infrastructure.Import/Services/Importer.cs

```diff
@@ -107,7 +107,10 @@ public IDataSourceFile LoadFile(IReadOnlyList<ColumnInfo> columnInfos, string fi
          {
             dataSource.AvailableFormats = CalculateFormat(dataSource, columnInfos, metaDataCategories, x).ToList();
             if (dataSource.AvailableFormats.Any())
+            {
+               dataSource.FormatCalculatedFrom = x;
```

Comment:
Let's name that variable something else. x is fine in small lambda stuff. But we are in a foreach loop here. I want a real name. THis is what I mean with constant refactoring. This should have been caught before and I missed it. But it's never to late:)